src/Illuminate/Database/Query/Grammars/SQLiteGrammar.php

```diff
@@ -193,6 +193,12 @@ protected function compileUpdateWithJoinsOrLimit(Builder $query, array $values)
      */
     public function prepareBindingsForUpdate(array $bindings, array $values)
     {
+        $values = collect($values)->reject(function ($value, $column) {
```

Comment:
Please remove the `reject()` clause. It's not relevant until we support JSON UPDATE queries on SQLite (i.e. `->update(['foo->bar' => 'baz'])`).

Reply:
Ah right, thanks for that, I wasn't sure why it was there, the postgres grammer was built differently so a comparison was hard.